server/config/routes.js

```diff
@@ -379,6 +379,7 @@ exports.configure = function configure(app) {
 
   app.get('/patients/:uuid/visits', patients.checkin.list);
   app.post('/patients/:uuid/checkin', patients.checkin.create);
+  app.get('/patients/:uuid/invoices/latest', patients.checkin.latest);
```

Comment:
Hmmm...  `patients.checkin.latest` makes it seem like the latest patient checkin.  Could you put this code somewhere else?   For example, in `patientInvoices.latest` or `patients.latestInvoice`?